recipes/harfbuzz/all/conandata.yml

```diff
@@ -30,3 +33,6 @@ patches:
   "2.7.4":
     - patch_file: "patches/source_subfolder.patch"
       base_path: "."
+  "2.8.0":
+    - patch_file: "patches/source_subfolder_2.8.X.patch"
```

Comment:
maybe call the file "fix_cmake_find..."

Reply:
I think it's better to keep a coherent name with the other patch, because it is exactly the same change (only the line number changed). Is it ok if I rename both patches ?